components/brave_vpn/resources/panel/components/contact-support/index.tsx

```diff
@@ -0,0 +1,234 @@
+                            ContactSupportToggleFields
+
+const defaultSupportState: ContactSupportState = {
+  contactEmail: '',
+  problemSubject: '',
+  problemBody: '',
+  shareHostname: true,
+  shareAppVersion: true,
+  shareOsVersion: true
+}
+
+type FormElement = HTMLInputElement | HTMLTextAreaElement | HTMLSelectElement
+type BaseType = string | number | React.FormEvent<FormElement>
+
+function ContactSupport (props: Props) {
+  const [supportData, setSupportData] = React.useState<BraveVPN.SupportData>()
+  const [formData, setFormData] = React.useState<ContactSupportState>(defaultSupportState)
+  const [showErrors, setShowErrors] = React.useState(false)
+  // Undefined for never sent, true for is sending, false for has completed
+  const [isSubmitting, setIsSubmitting] = React.useState<boolean>()
+  const [, setRemoteSubmissionError] = React.useState<string>()
+
+  // Get possible values to submit
+  React.useEffect(() => {
+    getPanelBrowserAPI().serviceHandler.getSupportData()
+      .then(setSupportData)
+  }, [])
+
+  function getOnChangeField<T extends BaseType = BaseType> (key: keyof ContactSupportInputFields) {
+    return function (e: T) {
+      console.log(setFormData)
+      const value = (typeof e === 'string' || typeof e === 'number') ? e : e.currentTarget.value
+      if (formData[key] === value) {
+        return
+      }
+      setFormData(data => ({
+        ...data,
+        [key]: value
+      }))
+    }
+  }
+
+  function getOnChangeToggle (key: keyof ContactSupportToggleFields) {
+    return function (isOn: boolean) {
+      if (formData[key] === isOn) {
+        return
+      }
+      setFormData(data => ({
+        ...data,
+        [key]: isOn
+      }))
+    }
+  }
+
+  const isValid = React.useMemo(() => {
+    return !supportData ||
+      !!formData.problemBody ||
+      !!formData.contactEmail ||
+      !!formData.problemSubject
+  }, [formData, supportData])
+
+  const handleSubmit = async () => {
+    // Clear error about last submission
+    setRemoteSubmissionError(undefined)
+    // Handle submission when not valid, show user
+    // which fields are required
+    if (!isValid) {
+      setShowErrors(true)
+      return
+    }
+    // Handle is valid, submit data
+    setIsSubmitting(true)
+    const fullIssueBody = formData.problemBody + '\n' +
+      (formData.shareOsVersion ? `OS: ${supportData?.osVersion}\n` : '') +
+      (formData.shareAppVersion ? `App version: ${supportData?.appVersion}\n` : '') +
+      (formData.shareHostname ? `Hostname: ${supportData?.hostname}\n` : '')
+
+    // TODO: this will return a bool (success) and string (message)
+    await getPanelBrowserAPI().serviceHandler.createSupportTicket(
+      formData.contactEmail,
+      formData.problemSubject,
+      fullIssueBody
+    )
+
+    // TODO: handle error case, if any?
+    setIsSubmitting(false)
+  }
+
+  const handlePrivacyPolicyClick = () => {
+    chrome.tabs.create({ url: 'https://brave.com/privacy/browser/#vpn' })
```

Comment:
i think this needs to be updated (or at least the privacy policy page itself needs to update) since it's linking to info about the iOS VPN

Reply:
Shared with @mattmcalister - we should be able to update the website 👍